src/plugins/credential_management/public/components/edit_credential/edit_credential.tsx

```diff
@@ -25,7 +25,7 @@ import {
   EuiConfirmModal,
 } from '@elastic/eui';
 import { DocLinksStart } from 'src/core/public';
-import { Credential } from '../../../../data_source/common';
```

Comment:
do we move Credential type from `common` to `public` ?

Reply:
The Credential type is still defined in `common` path, but because the CM plugin now depends on the DS plugin, when importing a non-public component, it will report the following error.

```
 ERROR in ./public/components/create_credential_wizard/create_credential_wizard.tsx
       │          Module not found: Error: import [../../../../data_source/common] references a non-public export of the [dataSource] bundle and must point to one of the public directories: [public]
       │           @ ./public/components/create_credential_wizard/create_credential_wizard.tsx 11:0-60 89:13-23
       │           @ ./public/components/create_credential_wizard/index.tsx
       │           @ ./public/components/index.tsx
       │           @ ./public/management_app/mount_management_section.tsx
       │           @ ./public/management_app/index.tsx
       │           @ ./public/plugin.ts
       │           @ ./public/index.ts
       │           @ /Volumes/workplace/DEV/packages/osd-optimizer/target/worker/entry_point_creator.js
```